src/main/resources/edu/cornell/kfs/coa/businessobject/datadictionary/AppropriationAccount.xml

```diff
@@ -1,164 +1,147 @@
 
-  <bean id="AppropriationAccount-inquiryDefinition-parentBean" abstract="true" parent="InquiryDefinition">
-    <property name="title" value="Sub-Fund Program Inquiry"/>
-    <property name="inquirySections">
-      <list>
-        <bean parent="InquirySectionDefinition">
-          <property name="title" value=""/>
-          <property name="numberOfColumns" value="1"/>
-          <property name="inquiryFields">
+    <!-- Business Object Inquiry Definition -->
+    
+    <bean id="AppropriationAccount-sectionDefinition" parent="AppropriationAccount-sectionDefinition-parentBean"/>
+    <bean id="AppropriationAccount-sectionDefinition-parentBean" parent="sectionDefinition" abstract="true" p:title="">
+        <property name="fields">
             <list>
-              <bean parent="FieldDefinition" p:attributeName="appropriationAccountNumber"/>
-              <bean parent="FieldDefinition" p:attributeName="appropriationAccountName"/>
-              <bean parent="FieldDefinition" p:attributeName="subFundGroupCode"/>
-              <bean parent="FieldDefinition" p:attributeName="projectNumber"/>
-              <bean parent="FieldDefinition" p:attributeName="active"/>
+                <ref bean="AppropriationAccount-appropriationAccountNumber"/>
+                <ref bean="AppropriationAccount-appropriationAccountName"/>
+                <ref bean="AppropriationAccount-subFundGroupCode"/>
+                <ref bean="AppropriationAccount-projectNumber"/>
+                <ref bean="AppropriationAccount-active"/>
             </list>
-          </property>
-        </bean>
-      </list>
-    </property>
-  </bean>
-
-<!-- Business Object Lookup Definition -->
+        </property>
+    </bean>
+    
+    <bean id="AppropriationAccount-inquiryDefinition" parent="AppropriationAccount-inquiryDefinition-parentBean"/>
+    <bean id="AppropriationAccount-inquiryDefinition-parentBean" abstract="true" parent="InquiryDefinition"
+          p:title="Sub-Fund Program Inquiry">
+        <property name="sections">
+            <list>
+                <ref bean="AppropriationAccount-sectionDefinition"/>
+            </list>
+        </property>
+    </bean>
 
 
-  <bean id="AppropriationAccount-lookupDefinition" parent="AppropriationAccount-lookupDefinition-parentBean"/>
+    <!-- Business Object Lookup Definition -->
 
-  <bean id="AppropriationAccount-lookupDefinition-parentBean" abstract="true" parent="LookupDefinition">
-    <property name="title" value="Appropriation Account Lookup"/>
-    
-    <property name="defaultSort">
-      <bean parent="SortDefinition">
-        <property name="attributeNames">
-          <list>
-            <value>appropriationAccountNumber</value>
-          </list>
+    <bean id="AppropriationAccount-lookupDefinition" parent="AppropriationAccount-lookupDefinition-parentBean"/>
+    <bean id="AppropriationAccount-lookupDefinition-parentBean" abstract="true" parent="AppropriationAccount-lookupDefinition-base-parentBean"/>
```

Comment:
Upon further review, I noticed you added another parent bean layer to the Appropriation Account lookup, likely as a result of following the setup for the Account lookup. I'm okay with this moving forward as such since I didn't find anything else in my second review that needs attention. In the future, though, we may want to remove the third bean if we don't intend to override this lookup's setup in our other module configs.